.travis.yml

```diff
@@ -1,17 +1,8 @@
 language: python
 env:
-   - DJANGO='>=1.8,<1.9'
-   - DJANGO='>=1.9,<1.10'
-   - DJANGO='>=1.10,<1.11'
-   - DJANGO='>=1.11,<1.12'
-   - DJANGO='>=2.0,<2.1'
+   - DJANGO='==2.1.5'
```

Comment:
Since Django 2.2 is already out, I'd consider adding `DJANGO='>=2.2,<2.3` to save us another ticket. By consequence, `requirements.txt` must then contain `Django>=2.1.5,<2.3` to signal to pip that it's compatible with Django 2.2.x.

Reply:
@pieterdd amended it